test_webhook.py

```
@@ -0,0 +1,80 @@
+    except requests.exceptions.ConnectionError:
+        print("❌ Webhook server not running. Start it with: python webhook_server.py")
+        return False
+    
+    # Test record start endpoint
+    payload = {
+        "story_id": "story_20250827_pr34",
+        "date": "2025-08-27"
+    }
+    
+    try:
+        response = requests.post(
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)